01-ruby-fundamentals/intro-to-object-types.md

```diff
@@ -197,13 +199,17 @@ ada.length
 # => 4
 ```
 
+
+
+
+
 Look at the Ruby docs for [Array](http://www.ruby-doc.org/core-2.4.0/Array.html) and [Hash](http://www.ruby-doc.org/core-2.4.0/Hash.html) for more details about collections.
```

Comment:
We should update these links to not reference the specific version but reference the latest by just using `/core/` instead